### PR TITLE
Re-adds the warden's compact shotgun

### PR DESCRIPTION
**Pull request by Kaffe-work**
## About The Pull Request

This PR re-adds the warden's compact shotgun, but also makes it slower to shoot (same speed as a riot shotgun, but without manual reloading)

## Why It's Good For The Game

Lethaling tiders, traitors and nuclear operatives in armory is your job. 

## Changelog
:cl:IradT
add: The compact shotgun has been re-issued to wardens on the station after some adjustments
balance: The compact shotgun fires a little slower now
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 1]
```diff
@@ -63,6 +63,16 @@
 	fire_delay = 5
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/com
 	w_class = WEIGHT_CLASS_HUGE
+//Warden Compact Shotgun
+
+/obj/item/gun/ballistic/shotgun/automatic/combat/compact
```
Comment (bobbahbrown) [inline code/modules/projectiles/guns/ballistic/shotgun.dm:68]:
```suggestion

//Warden Compact Shotgun
/obj/item/gun/ballistic/shotgun/automatic/combat/compact
```

**Comment by optimumtact** [comment]
I cast disappear

**Comment by optimumtact** [comment]
We removed this for good reason, and I'm not introduced in adding more shotguns that are not bulky and fit in the back slots.

Not until we come up with a good way to manage shotgun power creep.

**Comment by Kaffe-work** [comment]
So you are telling me if its bulky, i.e no backpack its fine?

**Comment by optimumtact** [comment]
no, because then it's just another shotgun

**Comment by Kaffe-work** [comment]
alright, I'll give warden a combat shotgun then :^)

**Comment by optimumtact** [comment]
you can try

**Comment by SkeletalElite** [comment]
The compact combat shotgun was always bulky. It's just that it's able to (just as riot shotguns are able to) be put in the armor slot, regardless of its weight class.
The combat shotgun is the only shotgun that can't be put in armors suit storage slot. That's why no one ever orders them.

**Comment by Kaffe-work** [comment]
i like to buy them to lethal people

**Pull request closed without merging**